### ISSUE TITLE
Pokemon Super Mystery Dungeon freezes

### ISSUE THREAD

**Issue by eskader119** [citra-emu/citra#5425]
The game freezes right before the personality test, after touching the touch-screen

**Comment by wwylele** [comment]
Log and version number please. Tbis sounds like a very old bug that should have been fixed.

**Comment by eskader119** [comment]
> Log and version number please. Tbis sounds like a very old bug that should have been fixed.

I know it's an old bug but for some reason but for some reason it's stil happening. I'm using canary 1791
[citra_log.txt](https://github.com/citra-emu/citra/files/4823304/citra_log.txt)

**Comment by wwylele** [comment]
Could you please test nightly?

**Comment by wwylele** [comment]
I tested it on Nightly 1580 and it seemed fine. Haven't tested canary yet

**Comment by sebas9916** [comment]
Hi, im having the same issue, the game just freezes right before the personality test. Im using the nightly 1601

**Comment by Dragios** [comment]
Something is definitely not right. It's working fine here
![Pokemon SMD](https://user-images.githubusercontent.com/18496868/89861797-2d24e400-dbd9-11ea-8707-105286309565.png)

What are your hardware specs?

**Comment by sebas9916** [comment]
My latop is an intel core i5-4210U, 4GB RAM, 64 bits system.

I tried to take a screenshot of the freezing issue, but for some reason it came out just black. Is it my laptop too shitty or is the rom that fails? Cause i´ve seen other people play the game

**Comment by Dragios** [comment]
I cannot say much for sure. I assume you are using the built in integrated Intel HD GPU? Does your laptop have dedicated GPU like nVidia or AMD?

**Comment by sebas9916** [comment]
No, it just has what it came with. I played other games like Pokemon Ultra Sun and Alpha Saphire and smash bros 3ds and nothing like this happened. Also, it freezes but the sound still goes on and the controllers work(?) like it sounds like im moving the options but the screen is just frozen.

**Comment by Dragios** [comment]
Which means purely Intel HD GPU. That could be the problem. AMD GPU too. Maybe it works last time and broken with recent commits with graphic code update in Citra. And it depends on games as each game performs vary.

**Comment by sebas9916** [comment]
so... what do i do then? do i have to change the graphics on the emulation or no matter what the game will just not run?

**Comment by Dragios** [comment]
I cannot say for sure as we need more confirmation, but if really due to hardware & driver that causes this, there is nothing you can do unfortunately as it requires hardware change. Let's just wait a developer to chime in and see what are their thoughts.

In the mean time, if you have time, maybe try to bisect which Citra version is broken on your side? You can start with Nightly 800 and if it works then go to Nightly 1200. Repeat this as you halve the version range and you should be able to find which version that starts to break.

**Comment by Davidy22** [comment]
Here to report a different issue that occurs playing super mystery dungeon, but saw this come up when searching existing issues. I can confirm that this bug did happen for me, but only when emulation is sped up. At 100% speed, I am able to get past the opening. I was using 300%, can consistently reproduce both 100% working and 300% failing.
